doc/security/vulnerabilities.rst

```diff
@@ -690,3 +721,22 @@ infinite loop, resulting in a denial of service attack.
 
 - `PR26571 fix for v2.4
   <https://github.com/zephyrproject-rtos/zephyr/pull/26571>`_
+
+
+CVE-2021
+========
+
+CVE-2021-3320
+-------------------
+
+Under embargo until 2022-04-14
```

Comment:
same question here, is the year correct?

Reply:
Good catch. The typo is actually where this data came from. Fixed both. Thanks !